### PR TITLE
Update getVideoPlaybackQuality data for Chrome etc

### PR DESCRIPTION
**Pull request by sideshowbarker**
See https://bugs.chromium.org/p/chromium/issues/detail?id=644731#c47 and
https://web-confluence.appspot.com/#!/catalog?q=%22getVideoPlaybackQuality%22

cc @jpmedley for review

### CODE REVIEW

File: api/HTMLVideoElement.json [hunk 1]
```diff
@@ -52,7 +52,7 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/HTMLVideoElement/getVideoPlaybackQuality",
           "support": {
             "chrome": {
-              "version_added": "23"
+              "version_added": "80"
```
Comment (jpmedley) [inline api/HTMLVideoElement.json]: Chrome Android is missing.

Comment (a2sheppy) [reply api/HTMLVideoElement.json]: Also, Firefox removes `corruptedVideoFrames` in version 73.

Comment (sideshowbarker) [reply api/HTMLVideoElement.json]: > Chrome Android is missing.

oops — thanks for catching that; added in https://github.com/mdn/browser-compat-data/pull/5452/commits/cb48157

Comment (sideshowbarker) [reply api/HTMLVideoElement.json]: > Also, Firefox removes `corruptedVideoFrames` in version 73.

Thanks for the heads-up. https://github.com/mdn/browser-compat-data/pull/5452/commits/79558e4 adds `version_removed` for that and also makes other fixes/updates to the data — to reflect that `corruptedVideoFrames`:

* was never actually implemented in Chrome (as far as I can tell — at least not in any way exposed to devs; at https://github.com/w3c/media-playback-quality/pull/20#issuecomment-572848608 I’ve asked for confirmation)
* has been marked as deprecated in the spec https://w3c.github.io/media-playback-quality/#dom-videoplaybackquality-corruptedvideoframes

Comment (jpmedley) [reply api/HTMLVideoElement.json]: It shipped in 80. https://storage.googleapis.com/chromium-find-releases-static/6c2.html#6c2d80337cb1988e53ae29f7418b52e9d81dec8e

If it hadn't, I would have asked that the version number be removed.

Comment (sideshowbarker) [reply api/HTMLVideoElement.json]: >> `corruptedVideoFrames`
>
> It shipped in 80. https://storage.googleapis.com/chromium-find-releases-static/6c2.html#6c2d80337cb1988e53ae29f7418b52e9d81dec8e

OK, thanks — pushed 0d19769 to update that

**Comment by ddbeck** [comment]
@sideshowbarker I've labeled this as rebase needed, but it also needs the attention for the linter failure. Thank you for your help on this!

File: api/VideoPlaybackQuality.json [hunk 2]
```diff
@@ -89,10 +93,10 @@
               "notes": "Only works on Windows 8+."
             },
             "opera": {
-              "version_added": "15"
+              "version_added": "67"
             },
             "opera_android": {
-              "version_added": "14"
+              "version_added": "67"
```
Comment (vinyldarkscratch) [inline api/VideoPlaybackQuality.json]: Looks like this should also be set to “false”.

Comment (sideshowbarker) [reply api/VideoPlaybackQuality.json]:
```
             "opera_android": {
-              "version_added": "14"
+              "version_added": "67"
```
> Looks like this should also be set to “false”.

OK, thanks — made that change

**Comment by sideshowbarker** [comment]
> I've labeled this as rebase needed, but it also needs the attention for the linter failure

Thanks — rebased and now no more linter failure

**Pull request merged**